lib/devcycle-ruby-server-sdk/api/dev_cycle_provider.rb

```diff
@@ -32,7 +32,16 @@ def fetch_number_value(flag_key:, default_value:, evaluation_context: nil)
     end
 
     def fetch_integer_value(flag_key:, default_value:, evaluation_context: nil)
-      @client.variable(Provider.user_from_openfeature_context(evaluation_context), flag_key, default_value)
+      variable = @client.variable(Provider.user_from_openfeature_context(evaluation_context), flag_key, default_value)
+
+      Variable.new(
```

Comment:
@kaushalkapasi I actually question why we aren't using the proper OpenFeature resolution details as we do in all other SDKs, but that's probably a separate PR that we should make.